### ISSUE TITLE
Weapon idea: Vintage folding shotgun

### ISSUE THREAD

**Issue by randomtyper** [CleverRaven/Cataclysm-DDA#66027]
### Is your feature request related to a problem? Please describe.

There was a plenty of rare, cool and wacky firearm designs in history, yet there is not many true rare weapons outside of secret lab and advanced military designs. Many such designs are very normal (like M1897, your typical pump shotgun, even if with a nasty bayonet) or are pretty well-known (LeMat, cap-and-ball Colts). I think there should be something even rarer, something that a real collectioner will give both of his kidneys away for, yet useful for a survivalist.

Enter the Burgess folding shotgun. A weapon with a 20 inch barrel that can fit into almost any large backpack in folded state, yet takes only a vigorous wrist flick to be ready to fire. Rare, with production numbers in low thousands (similar to LeMat in that regard), this weapon may come with a proprietary holster that can be attached to your leg or MOLLE (with some modification, most likely).

I hope this will evolve into something more extensive, like a Forgotten Weapons project or something, with a very rare manual (that gives some decent joy, too) or a flash drive/SD card with data on exotic firearms. You would be impressed to learn what interesting designs were made in past, like metallic cartridges for flint/wheellocks as early as 1600s (or even 1500s) or multishot light flintlock revolver cannons in 1700s. Something a knowledgeable and skilled survivor would certainly take advantage of, if better options are absent. Some early breechloaders may even be possible to craft in survival conditions, too, such as Sharps. And, when multiple weapon containers will be a thing, maybe antique airguns will make an entry, possibly even in craftable format.

### Solution you would like.

A new shotgun that is mostly a copy of M1897, with exception of having lower durability (6-7 is enough, it is an old sturdy gun, but less sturdy than the trench broom), 5 full-size rounds in mag (it could fit 6 2.5 inch shells, though), no grip mount (grip has a part of the reloading mechanism) and is about 500 mm long (its folded state, from which it can be easily readied) and a bit smaller in volume.

Spawns in mansions and musems (maybe not, but I think it is rare and notable enough to be bought into some collections), and/or where M1997 and LeMat do, but more rare. Has a high chance to spawn with a custom belt holster, though that holster will maybe have an autolearn recipe. 

Basically a very rare pocket shotgun that can be packed into a backpack and can behave not unlike a pistol when used with its holster. A neat and (imo) practical weapon that is also a good target to obtain for collectors and hoarders.

### Describe alternatives you have considered.

Not adding a very cool and potentially useful for survivors piece of American weaponmaking history.

### Additional context

Articles:
1. https://surplused.com/index.php/2020/03/22/the-burgess-shotguns/
2. https://www.forgottenweapons.com/burgess-folding-shotgun/

Video on how it works:
https://www.youtube.com/watch?v=HXvmGtLYwKA

**Comment by TheShadowFerret** [comment]
>The shotguns are fairly rare today, with folding models particularly hard to find – and the rifles are extremely scarce. In 1899, Burgess decided to retire, and sold his company and factory to the Winchester company. They promptly repurposed what tooling and machinery they could, scrapped the rest, and shut down the factory.

This is too rare to include, & there are no reproduction makers of it like the LeMat(reproductions made by Pietta).

**Comment by randomtyper** [comment]
> > The shotguns are fairly rare today, with folding models particularly hard to find – and the rifles are extremely scarce. In 1899, Burgess decided to retire, and sold his company and factory to the Winchester company. They promptly repurposed what tooling and machinery they could, scrapped the rest, and shut down the factory.
> 
> This is too rare to include, & there are no reproduction makers of it like the LeMat(reproductions made by Pietta).

So, absolutely no chance one or two found their way to New England? Even with chance of 1 per 40 LeMats? That's a shame. I thought it would be neat to have it for that "oh, now that is a really rare thing". Though I guess it won't be that much rarer than ammonia machine crafting materials (with how many of those you need and how scarce they are, I wonder why is it even in game). That said, with some lifted tooling it may not be impossible to reproduce one... Modern machinery came a long way from 1890s, after all.

**Comment by randomtyper** [comment]
Closing the issue, not much work on this right now, but it may resurface at some point.
